src/Microsoft.DocAsCode.Metadata.ManagedReference/Transform/TripleSlashCommentTransform.xsl

```diff
@@ -111,6 +111,34 @@
     </xsl:if>
   </xsl:template>
 
+  <xsl:template match="note">
+    <xsl:variable name="type">
+      <xsl:value-of select="normalize-space(@type)"/>
```

Comment:
if so, update line 133 to lowercase the "Note" to "note", to be consistent with lowercase $type?

---
In reply to: [95945168](https://github.com/dotnet/docfx/pull/1171#discussion_r95945168) [](ancestors = 95945168)